kolena/_api/v1/event.py

```diff
@@ -64,6 +64,7 @@ class Event(str, Enum):
         FETCH_DATASET_MODEL_RESULT = "sdk-dataset-model-result-fetched"
         UPLOAD_DATASET_MODEL_RESULT = "sdk-dataset-model-result-uploaded"
         FETCH_DATASET_MODEL_RESULT_BY_TAG = "sdk-dataset-model-result-fetched-by-tag"
+        GET_MODELS_BY_DATASET = "sdk-dataset-models-fetched-by-dataset"
```

Comment:
Named slightly differently from those above to distinguish downloading results vs listing the models.